### PR TITLE
PageAsyncMultiSelect & PageAsyncSingleSelect

### PR DESCRIPTION
**Pull request by jamestalton**
`PageAsyncMultiSelect` wraps the `PageMultiSelect` component to support asynchronously loading options from and async source such as an API.

### CODE REVIEW

File: framework/PageInputs/PageAsyncMultiSelect.cy.tsx
Comment (nixocio) [inline framework/PageInputs/PageAsyncMultiSelect.cy.tsx:1]: Very good tests. Helpful to understand how the component works.

**Review by nixocio — APPROVED** [review]
Hey @jamestalton great job. I left minor comments. The tests were really helpful to allow to verify the components behavior.

File: framework/PageInputs/PageInputs.md [hunk 1]
```diff
@@ -1,33 +1,33 @@
...
 The base components are wrappers over the PatternFly components with a goal to make the component interface simpler.
 
 - `PageSingleSelect`: A select dropdown component that supports single selection of options.
 - `PageMultiSelect`: A select dropdown component that supports multiple selection of options.
 
-## Async Components
+### Async Select Components
 
-The async components are wrappers over the base components with a goal of loading options asynchrously from APIs and supporting pagination of results.
+The async components are wrappers over the base seelcts components with a goal of loading options asynchrously from APIs and supporting pagination of results.
```
[the remark this answers is not in the document]
Comment (jamestalton) [reply framework/PageInputs/PageInputs.md:12]: fixed thanks

File: framework/PageInputs/PageMultiSelect.tsx [hunk 2]
```diff
@@ -105,6 +135,8 @@ export function PageMultiSelect<ValueT>(props: {
     setSearchValue('');
     if (isOpen && searchRef.current) {
       searchRef.current.focus();
+    } else {
+      // selectRef.current?.focus();
```
[the remark this answers is not in the document]
Comment (jamestalton) [reply framework/PageInputs/PageMultiSelect.tsx:139]: removed

**Pull request merged**